Base/usr/share/man/man4/audio.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+audio - system audio device
+
+## Description
+
+The `/dev/audio` character device file exposes the audio output device of the system. As of now, this is an output-only device and reading it has no effect. To get the audio device to play audio, PCM samples need to be written to `/dev/audio` as a series of "frames" (in MPEG speech) or multi-channel samples with the following format:
```

Comment:
`in MPEG speech` seems like a weird word choice. Maybe `in MPEG terminology` instead?